charts/bandstand-web-service/templates/tests/pod.yaml

```diff
@@ -51,6 +51,11 @@ spec:
         {{- if (.Values.test).environmentEnvVars }}
           {{- .Values.test.environmentEnvVars | toYaml | trimPrefix "|" | trimPrefix "\n" | nindent 8 }}
         {{- end }}
+      envFrom:
+        {{- if .Values.sharedEnvConfig }}
+        {{- .Values.sharedEnvConfig | toYaml | trimPrefix "|" | trimPrefix "\n" | nindent 12 }}
+        {{- end }}
```

Comment:
```suggestion
      {{- if .Values.sharedEnvConfig }}
      envFrom:
        {{- .Values.sharedEnvConfig | toYaml | trimPrefix "|" | trimPrefix "\n" | nindent 12 }}
      {{- end }}
```

Reply:
thanks, was just looking at it and forgot to update the name and use the test too 🤦